merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/mocks/StubMissionModelService.java

```diff
@@ -50,6 +51,8 @@ public final class StubMissionModelService implements MissionModelService {
       NONEXISTENT_ACTIVITY_TYPE,
       Map.of());
 
+  public static final Map<String, ValueSchema> RESOURCES;
```

Comment:
I think it would be a bit more idiomatic to use the `Map.of` factory method here.

```suggestion
  public static final Map<String, ValueSchema> RESOURCES = Map.of(
    "mode", ValueSchema.ofVariant(List.of(
        new ValueSchema.Variant("Option1", "Option1"),
        new ValueSchema.Variant("Option2", "Option2")
    )),
    "state of charge", ValueSchema.REAL
  );
```

Reply:
done

Reply:
done

Reply:
Actually, after implementing the change, the code gen test fails 50% of the time. Apparently `Map.of` does not maintain a deterministic order. HashMap does (it doesn't keep insertion order, it keeps hash order), so it worked by coincidence.

Reply:
Ah, dang - why does the order matter? Is it because we're doing string matching on generated code?

Reply:
exactly. sometimes the resources are inserted into the typescript code in a different order, which makes the CodeGen test complain, because it's just checking for exact string equality.

Reply:
As part of reverting, I changed it to a `LinkedHashMap`, which preserves insertion order.